### PR TITLE
Removed ServiceAccounts

### PR DESCRIPTION
**Pull request by pawicao-ibm**
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/47934

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -322,8 +322,6 @@ prepare-unit-test:
 	kubectl create secret generic artifactory-token -n ${NAMESPACE} --from-file=.dockerconfigjson=./artifactory.yaml --type=kubernetes.io/dockerconfigjson || echo ""
 	kubectl apply -f ./config/crd/bases/operator.ibm.com_ibmlicenseservicereporters.yaml || echo ""
 	kubectl apply -f ./config/crd/bases/operator.ibm.com_ibmlicensings.yaml || echo ""
-	kubectl apply -f ./bundle/manifests/ibm-license-service_v1_serviceaccount.yaml -n ${NAMESPACE} || echo ""
```
Comment (kacperkrzyzak) [inline Makefile:325]: We need to replace old sa, not only delete it

File: docs/Content/Install_offline.md [hunk 2]
```diff
@@ -119,8 +119,6 @@ kubectl apply -f config/crd/bases/operator.ibm.com_ibmlicensings.yaml
 kubectl apply -f config/crd/bases/operator.ibm.com_ibmlicenseservicereporters.yaml
 # add RBAC:
 kubectl apply -f config/rbac/role.yaml
-kubectl apply -f bundle/manifests/ibm-license-service_v1_serviceaccount.yaml
```
Comment (kacperkrzyzak) [inline docs/Content/Install_offline.md:122]: We need to replace old sa, not only delete it

Comment (pawicao-ibm) [reply docs/Content/Install_offline.md:122]: Done

File: Makefile (hunk 1 above)
Comment (pawicao-ibm) [reply Makefile:325]: Done

**Review by kacperkrzyzak — APPROVED** [review]

**Pull request merged**